packages/headless/src/controllers/commerce/recommendations/headless-recommendations.ts

```diff
@@ -0,0 +1,118 @@
+import {Schema} from '@coveo/bueno';
+import {createSelector} from '@reduxjs/toolkit';
+import {CommerceAPIErrorStatusResponse} from '../../../api/commerce/commerce-api-error-response';
+import {Product} from '../../../api/commerce/common/product';
+import {
+  CommerceEngine,
+  CommerceEngineState,
+} from '../../../app/commerce-engine/commerce-engine';
+import {configuration} from '../../../app/common-reducers';
+import {contextReducer as commerceContext} from '../../../features/commerce/context/context-slice';
+import {recommendationSlotDefinition} from '../../../features/commerce/recommendation/recommendation';
+import {
+  fetchRecommendations,
+  updateRecommendationSlotId,
+} from '../../../features/commerce/recommendation/recommendation-actions';
+import {recommendationV2Reducer as recommendation} from '../../../features/commerce/recommendation/recommendation-slice';
+import {loadReducerError} from '../../../utils/errors';
+import {validateInitialState} from '../../../utils/validate-payload';
+import {
+  buildController,
+  Controller,
+} from '../../controller/headless-controller';
+
+/**
+ * The `Recommendations` controller exposes a method for retrieving recomendations content in a commerce interface.
+ */
+export interface Recommendation extends Controller {
```

Comment:
Whatever we settle on, we should be consistent
```suggestion
 * The `Recommendation` controller exposes a method for retrieving recomendations content in a commerce interface.
 */
export interface Recommendation extends Controller {
```